server/main.ts

```diff
@@ -36,7 +37,15 @@ serve(async (req) => {
     });
   }
 
-  if (pathname === "/" || pathname === "/it" || pathname === "/en") {
+  let lang: keyof typeof strings | null = null;
+  for (const langId of Object.keys(strings)) {
+    if (pathname === `/${langId}`) {
+      lang = langId as keyof typeof strings;
+      break;
+    }
+  }
+
+  if (pathname === "/" || lang !== null) {
```

Comment:
```suggestion
  const validLangs = Object.keys(strings) as Array<keyof typeof strings>;
  const lang = validLangs.find(lang => pathname === `/${lang}`);

  if (pathname === "/" || lang !== undefined) {
```

I didn't test this, but the loop looks exactly like what `.find` does.

Reply:
Right! :man_facepalming: